HanekeTests/NSFileManager+HanekeTests.swift

```diff
@@ -21,7 +21,7 @@ class NSFileManager_HanekeTests: DiskTestCase {
     
     func testEnumerateContentsOfDirectoryAtPathStop() {
         let sut = FileManager.default
-        [self.writeDataWithLength(1), self.writeDataWithLength(2)]
+        _ = [self.writeDataWithLength(1), self.writeDataWithLength(2)]
```

Comment:
This was a warning in Xcode. I simply put this

```
_ = 
```

The test itself is not mine. It was there from the outset.

Reply:
This doesn't change the fact that it's not needed. if you do house keeping I would expect these things tidied up, or not changed at all.

Reply:
Go fuck yourself

Reply:
XCTAssertNotNil

Reply:
Man, if you want to make contributions accept code reviews. This behaviour is highly unprofessional.

Reply:
I'm not a professional. You are the rudest cunt I have ever met. Nothing more.

Reply:
@rob-nash this is just a code review. I've always been polite. You can accept critics and improve, or you can behave like that. The point is that what I'm saying is that in this particular scope this code is not optimal and can be optimised. A good approach would be asking how rather than sending me to fuck off. and I would gladly explain that the array is not contributing to the test, therefore can be separated in 2 instructions, tiding the code a little bit.